package.json

```diff
@@ -43,7 +43,7 @@
   },
   "homepage": "https://github.com/strongloop/fsevents",
   "devDependencies": {
-    "node-gyp": "^3.8.0",
+    "node-gyp": "*",
```

Comment:
Just don't depend on node-gyp at all. It's always bundled with npm and that's always the version you should use. :-)